fearless/Modules/Staking/Model/StakingCallFactory.swift

```diff
@@ -6,7 +6,7 @@ import BigInt
 protocol SubstrateCallFactoryProtocol {
     func bond(amount: BigUInt,
               controller: String,
-              rewardDestination: RewardDestination) throws -> RuntimeCall<BondCall>
+              rewardDestination: RewardDestination<String>) throws -> RuntimeCall<BondCall>
```

Comment:
so that `RewardDestination<AccountId>`

Reply:
now it is RewardDestination<AccountAddress>